linebot/exceptions.py

```diff
@@ -58,22 +58,24 @@ def __init__(self, message='-'):
 class LineBotApiError(BaseError):
     """When LINE Messaging API response error, this error will be raised."""
 
-    def __init__(self, status_code, error=None):
+    def __init__(self, status_code, request_id=None, error=None):
         """__init__ method.
 
         :param int status_code: http status code
+        :param str request_id: (optional) Request ID. A unique ID is generated for each request
```

Comment:
Would you like to add response headers at this time?
In case of error, it is better to have as much information as possible.

Reply:
I agree with your opinion.